contrib/inventory/ec2.py

```diff
@@ -672,7 +672,9 @@ def get_elasticache_clusters_by_region(self, region):
                 error = self.get_auth_error_message()
             if not e.reason == "Forbidden":
                 error = "Looks like AWS ElastiCache is down:\n%s" % e.message
-            self.fail_with_error(error, 'getting ElastiCache clusters')
+                self.fail_with_error(error, 'getting ElastiCache clusters')
+            else:
+                return
```

Comment:
This is precisely what I'm trying to avoid. It should be transparent to me - if I don't have permissions, I don't actually care, I don't want it to error out, I'm happy for the Elasticache results to be empty.